feelpp/feel/feeldiscr/timeset.hpp

```diff
@@ -679,9 +682,21 @@ class TimeSet
                     }
                     else
                     {
-                        M_ts->M_scalar_p1 = scalar_p1_space_type::New(_mesh=M_mesh.get(),
-                                                                      _worldscomm=func.worldsComm(),
-                                                                      _extended_doftable=std::vector<bool>(1,extendeddof) );
+                        if constexpr ( std::is_same_v<scalar_p1_space_type,typename FunctionType::functionspace_type> )
+                        {
+                            if ( ( func.mesh() == M_mesh ) && ( func.functionSpace()->basisOrder()[0] == 1 ) )
```

Comment:
@prudhomm in fact, we cannot apply this optimisation generally for the ensightgoldformat because it requires to have the extended dof table. Another check is necessary : 
```
if ( ... &&  func.functionSpace().extendedDofTable() == extendeddof )
```

Reply:
yes indeed :(

I am wondering  : is it indeed really necessary to use the extended dof in ensight ? we still have those interfaces we see in 3D

Reply:
if I remember, the extended doftable removes interface with Ensight software but not in Paraview.
Anyway, I do not fully agree to have this kind of space here, and I am sure that we can retrieve these ghosts information quite easily. Maybe we can remove these feature by default and try to re-add in the future  the  ghost information without the extended doftable